back-end/app.js

```diff
@@ -1,13 +1,25 @@
-// DEPENDENCIES
 
-const express = require("express");
 
-// CONFIGURATION
+const express = require('express');
+const cors = require('cors')
 const app = express();
 
-// MIDDLEWARE
 
-// ROUTES
+app.use(cors())
+app.use(express.json())
+
```

Comment:
Let's get rid of this extra whitespace - one line should be enough to break up our code